install/kubernetes/helm/istio/charts/security/templates/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  # we use the normal name here (e.g. 'prometheus')
+  # as grafana is configured to use this as a data source
+  name: istio-ca
+  labels:
+    app: istio-ca
+spec:
+  type: LoadBalancer
```

Comment:
do we always want to do loadbalancer type for this?  I asked as istio ingress has a node port option.

Reply:
That's what we have in istio.yaml - we can add a node port as well, but separate PR :-)